Makefile

```diff
@@ -31,13 +31,27 @@ Makefile.inc: configure
 	@echo "$@ generated. Run the make again."
 	@exit 1
 
-install: Makefile.inc
+hwdata.pc: hwdata.pc.in
+	datadir="$(datadir)"; \
+	if [ "$${datadir#$(prefix)}" != "$$datadir" ]; then \
+		datadir="\$${prefix}$${datadir#$(prefix)}"; \
+	fi; \
+	sed -e 's|@prefix@|$(prefix)|g' \
+		-e "s|@datadir@|$$datadir|g" \
+		-e 's|@pkgdatadir@|$${datadir}/$(NAME)|g' \
+		-e 's|@VERSION@|$(VERSION)|g' \
+		-e 's|@NAME@|$(NAME)|g' \
+		hwdata.pc.in > hwdata.pc
+
+install: Makefile.inc hwdata.pc
 	mkdir -p -m 755 $(DESTDIR)$(datadir)/$(NAME)
 	for foo in $(IDFILES) ; do \
 		install -m 644 $$foo $(DESTDIR)$(datadir)/$(NAME) ;\
 	done
 	mkdir -p -m 755 $(DESTDIR)$(libdir)/modprobe.d
 	install -m 644 -T blacklist.conf $(DESTDIR)$(libdir)/modprobe.d/dist-blacklist.conf
+	mkdir -p -m 755 $(DESTDIR)$(libdir)/pkgconfig
+	cp -f hwdata.pc $(DESTDIR)$(libdir)/pkgconfig/
```

Comment:
since hwdata contains only data files, the pkg-config file for it does not need to be in the arch-specific library path; `$datadir/pkgconfig` is used by pkg-config implementations too for such kind of "packages"

hence, i'd say this ought to be installed to `$(datadir)/pkgconfig`

Reply:
Good idea, thanks.